extension/src/Package.ts

```diff
@@ -274,7 +274,7 @@ export class Package {
      * Call `updateState()` first to ensure this is up-to-date.
      */
     public get isUpdateAvailable(): boolean {
-        return !!this.installedVersion && this.version > this.installedVersion;
+        return !!this.installedVersion && this.version.compare(this.installedVersion) == 1;
```

Comment:
```suggestion
        return !!this.installedVersion && this.version.gt(this.installedVersion);
```
Comparison functions are usually expected to return a value `< 0`, `== 0`, or `> 0` instead of exactly -1, 0, or 1, so comparing against 1 might not be safe. https://github.com/npm/node-semver says there's a `gt()` function though, so I think we should just use that.

Reply:
SemVer class doesn't expose a `gt()` function : https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/semver/classes/semver.d.ts

Reply:
Oh, sorry, it's a free function `semver.gt()`, not a class function `SemVer.gt()`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/682c4f6e9039738cf16ca94393367c43ab27b82a/types/semver/index.d.ts#L59

Reply:
Sorry, I have an error "Uncaught ReferenceError: gt is not defined."
I don't understand why, I add an import for gt : `import { gt, parse as parseVersion, SemVer } from 'semver';`
and I call it : `return !!this.installedVersion && gt(this.version,this.installedVersion)`
Any idea ?